syntax = "proto3";

package NYql;

option java_package = "com.yandex.yql.proto";

message TSeverityIds {
    enum ESeverityId {
        S_FATAL = 0;
        S_ERROR = 1;
        S_WARNING = 2;
        S_INFO = 3;
    }
}
